dev_mode/index.js

```diff
@@ -162,20 +162,39 @@ function main() {
 
   // Handle a browser test.
   var browserTest = PageConfig.getOption('browserTest');
+  var el = document.createElement('div');
```

Comment:
@afshin This appears to be the cause of #5393. Is there a reason to add this div to the page with `'[]'`, and to do it outside of the if-statement?

Reply:
This is a mistake, I did not see realize this would have an  impact on the non-test environment. Sorry! I mistook this PR's consequences as only being server-side.

Reply:
Sounds good. Should this be moved inside the `if` statement, or removed entirely?